xarray/testing/strategies.py

```diff
@@ -192,6 +197,22 @@ def attrs() -> st.SearchStrategy[Mapping[Hashable, Any]]:
     )
 
 
+def serializable_attrs() -> st.SearchStrategy[Mapping[Hashable, Any]]:
```

Comment:
What makes the current attrs strategy produce non-serializable things? TBH that `attrs` strategy is the least important part of the existing `strategies` module, and generating nested things is probably taking time up for little coverage benefit, so I wouldn't be opposed to simplifying the main `attrs` strategy so you can use it here.